src/api/account/select.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Finds and returns an account based on given id.
+ *
+ * id - Can be an index of the array such as 0 or an address
+ **/
+module.exports = async (client, id) => {
+  // check for ethereum accounts and select a default one
+  const accounts = await client.web3.eth.getAccounts()
```

Comment:
@plutoegg Currently i'm automatically selecting accounts[0] but i have the impression that might to be ideal? Perhaps we should look for https://github.com/ethereum/wiki/wiki/JavaScript-API#web3ethdefaultaccount and if it doesn't exist we try accounts[0] ?